object-construction-checker/src/main/java/org/checkerframework/checker/objectconstruction/framework/AutoValueSupport.java

```diff
@@ -106,7 +102,11 @@ public void handlePossibleBuilderBuildMethod(AnnotatedExecutableType t) {
           getAutoValueRequiredProperties(nextEnclosingElement, avBuilderSetterNames);
       AnnotationMirror newCalledMethodsAnno =
           createCalledMethodsForAutoValueProperties(requiredProperties, avBuilderSetterNames);
-      t.getReceiverType().addAnnotation(newCalledMethodsAnno);
+      AnnotationMirror possibleBuildAnnotations =
```

Comment:
Add a code comment here indicated that we only add the new `@CalledMethods` annotation if there is not already a `@CalledMethods` annotation present